### PR TITLE
Adding php-cs-fixer

### PR DESCRIPTION
**Pull request by pdrillin**
* Ticket : #2 
*  Adding php-cs-fixer as Github action

### CODE REVIEW

File: .github/workflows/test-lint-php.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    name: "PHP-CS-Fixer"
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@master
+
+    - uses: docker://oskarstark/php-cs-fixer-ga
+      with:
+        args: --config=.php_cs.dist --allow-risky=yes --dry-run --diff
+
```
Comment (Gustry) [inline .github/workflows/test-lint-php.yml]: It should be an empty line at the end of a file.

**Review by Gustry — COMMENTED** [review]
Can you add the makefile commands as well ? 
I will let a PHP dev review this PR.

File: .github/workflows/test-lint-php.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  test-php-cs-fixer:
+    name: "PHP-CS-Fixer"
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@master
+
+    - uses: docker://oskarstark/php-cs-fixer-ga
+      with:
+        args: --config=.php_cs.dist --allow-risky=yes --dry-run --diff
```
Comment (rldhont) [inline .github/workflows/test-lint-php.yml]: It should be an empty line at the end of a file.

**Review by rldhont — CHANGES_REQUESTED** [review]
Just a quick fix before merge

**Review by rldhont — APPROVED** [review]

**Comment by mdouchin** [comment]
I manually ran the make command, and added more commits since this PR. @drillinP could you please rebase over master and push force this branch please ?

**Comment by pdrillin** [comment]
@mdouchin j'ai fais le rebase

**Comment by Gustry** [comment]
This has been done

**Pull request closed without merging**